bindings/nodejs/lib/types/output.ts

```diff
@@ -1,10 +1,16 @@
 // Copyright 2023 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-import { UnlockCondition } from './unlock_condition';
-import { Feature } from './feature';
+import {
+    UnlockCondition,
+    UnlockConditionDiscriminator,
+} from './unlock_condition';
+import { Feature, FeatureDiscriminator } from './feature';
 import { HexEncodedString, INativeToken, TokenSchemeTypes } from '@iota/types';
 
+// Temp solution for not double parsing JSON
```

Comment:
Why Temp?

Reply:
Ill fix completely in the merge pr, no point decorating half the classes now and deleting them next pr